src/otk_external_osbuild/command/make_depsolve_dnf4_rpm_stage.py

```diff
@@ -5,7 +5,14 @@
 
 def root(input_stream: TextIO) -> None:
     data = json.load(input_stream)
-    tree = data["tree"]["packageset"]["const"]["internal"]
+    tree = data["tree"]
+    pkgs = tree["packageset"]["const"]["internal"]
+
+    # allow passing on stage options
+    opts = tree.get("options", {})
```

Comment:
*maybe* `stage_options` or something (naming is hard) to indicate that this is passed straight to the rpm stage, i.e. that people can just lookup the meta.json to see what is available? But a bit on the fence, options is also nice and short.

Reply:
Something to consider in naming/format is that externals can of course generate multiple stages and it won't be entirely clear which stage the options would go to. I picked the `options` name but maybe we should be more explicit and provide explicit key names instead of passing it along (even if it's easy?).

Reply:
Oh, indeed, interesting, I wonder if we should nest then, like 
```
options:
 org.osbuild.rpm: 
   foo: bar
```
or just (because we know what stages are under the external)
```
options:
 rpm_stage:
...
````
but maybe more explicit is actually better here.

Reply:
I prefer the second option. I'll update this PR.

Reply:
Done. Split out the move of `gpgkeys` to #218 to not lose it.

Reply:
Cool, thanks! I think we still want a simple test and then this is good to go from my PoV